documentation/modules/con-entity-operator-logging.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-entity-operator.adoc
+
+[id='con-entity-operator-logging-{context}']
+= Operator loggers
+
+The Topic Operator and User Operator have a configurable logger:
+
+* `rootLogger.level`
+
+The operators use the Apache `log4j2` logger implementation.
+
+Use the `logging` property to configure loggers and logger levels.
+
+You can set the log levels by specifying the logger and level directly (inline) or use a custom (external) ConfigMap.
+If a ConfigMap is used, it must be placed under under the `log4j2.properties` key.
```

Comment:
typo, double "under"

Reply:
> If a ConfigMap is used, it must be placed under under the `log4j2.properties` key.

Tbh this statement could be misleading. It sounds to me that if you have a ConfigMap with the logging configuration, you have to put it (its name?) under the `log4j2.properties` key of the operator logging configuration.
Actually it is more like ...
In the `logging.name` property you have to set the name of the ConfigMap containing the external logging configuration. Inside the ConfigMap, you have to put the logging configuration under the `log4j2.properties`.

Reply:
Thanks Paolo. Much better. I will update each as: 

If a ConfigMap is used, you set `logging.name` property to the name of the ConfigMap containing the external logging configuration. Inside the ConfigMap, the logging configuration is described using `log4j2.properties`.